tests/integ/scala/test_update_delete_merge_suite.py

```diff
@@ -33,11 +33,7 @@
 table_name3 = Utils.random_name_for_temp_object(TempObjectType.TABLE)
 
 
-pytestmark = pytest.mark.xfail(
-    condition="config.getvalue('local_testing_mode')", raises=NotImplementedError
-)
-
-
+@pytest.mark.localtest
```

Comment:
We didn't check the column names and types in these live tests so no assertions on column types to the local test. How about adding some assertions?